x/params/circuit/ante.go

```diff
@@ -0,0 +1,22 @@
+package circuit
```

Comment:
If feasible, can we move the inclusion of circuit breaker stuff to a separate PR. This PR is already super big, and circuit breaker seems like it should be reviewed independently of the other changes that are going on.